ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_permissions_mgmt.robot

```diff
@@ -263,3 +268,18 @@ ${username} Should Not Have Access To ${project_title}
     Project Should Not Be Listed    project_title=${project_title}
     RoleBinding Should Not Exist    project_title=${project_title}
     ...    subject_name=${username}
+
+Assign ${permission_type} Permissions To User ${username} in Project ${project_title}
+    Open Data Science Projects Home Page
+    Wait Until Project Is Listed    project_title=${project_title}
+    Open Data Science Project Details Page    ${project_title}
+    Move To Tab             Permissions
+    Assign ${permission_type} Permissions To User ${username}
+
+Verify No Permissions For User ${username}
```

Comment:
since it is checking the user permission to dashboard, what do you think about:
1. renaming the kw to `User ${username} Should Not Be Allowed To Dashboard`
2. moving the keyword to `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot`

Reply:
I will change the name, but it's not a good idea to move this keyword to another resource since:
1. It is only used in this test suite
2. It is using `Switch To User` kw, which is unique to this suite, as it is actually switching to a new browser that was already defined for this user (user is actually browser ID, not user of RHODS)

Reply:
okay, thx